iree/compiler/Translation/SPIRV/XLAToSPIRV/test/adjust_integer_width.mlir

```diff
@@ -272,4 +272,26 @@ module{
       spv.Return
     }
   }
+
+  spv.module "Logical" "GLSL450" {
+    spv.globalVariable @globalInvocationID built_in("GlobalInvocationId") : !spv.ptr<vector<3xi32>, Input>
+    spv.globalVariable @constant_arg_0 bind(0, 0) : !spv.ptr<!spv.struct<i1 [0]>, StorageBuffer>
+    spv.globalVariable @constant_arg_1 bind(0, 1) : !spv.ptr<!spv.struct<i8 [0]>, StorageBuffer>
+    spv.func @select(%arg0 : i1, %arg1 : i8) -> () "None" {
+      %0 = spv._address_of @constant_arg_0 : !spv.ptr<!spv.struct<i1 [0]>, StorageBuffer>
+      %1 = spv.constant 0 : i32
+      %2 = spv.AccessChain %0[%1] : !spv.ptr<!spv.struct<i1 [0]>, StorageBuffer>
+      %3 = spv.Load "StorageBuffer" %2 : i1
+      // CHECK: spv.constant 0 : i32
+      // CHECK: spv.constant 1 : i32
```

Comment:
It's quite weird to check these two constant without capturing. In this test case, we are testing if spv.Select get adjusted. There are two options:

1) Check spv.Select itself.
2) Capture the values for spv.constant, and also for loading value, then check spv.Select with them.

For option two, more details can be found at https://mlir.llvm.org/getting_started/TestingGuide/